### PR TITLE
linting added

### LINKED ISSUE #583: Linting check
- [ ] Setup Github actions to check the linting

### PR DESCRIPTION
**Pull request by arunavabasu-03**
linting fixes #583

### CODE REVIEW

**Comment by suzil** [comment]
For the issue with the png, I think you could add the file `website/src/radis.d.ts` with the contents:
```ts
declare module "*.png";
```
Since `.gitignore` ignores these files, you could make an exception for this particular file with `!website/src/radis.d.ts` (or something like that).

**Comment by arunavabasu-03** [comment]
### 🌱 solutions that I follow -
https://stackoverflow.com/questions/57127606/ts2307-cannot-find-module-images-logo-png
https://www.codegrepper.com/code-examples/javascript/typescript+cannot+find+module+png

File: website/radis.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// declare module "*.jpg" {
+//   const value: string;
+//   export = value;
+// }
+// declare module "*.png" {
+//   const value: string;
+//   export = value;
+// }
+declare module "*.png";
```
Comment (suzil) [inline website/radis.d.ts:9]: I think this needs to be in the src/ directory but let me know otherwise.

**Review by suzil — APPROVED** [review]
Looks good! Thanks for the PR.

**Pull request merged**